wire-ios-data-model/Source/Model/Message/ZMAssetClientMessage.swift

```diff
@@ -175,10 +175,14 @@ import Foundation
 
     public override func expire() {
         super.expire()
+        WireLogger.assets.info("expire asset message", attributes: [LogAttributesKey.nonce.rawValue: self.underlyingMessage?.messageID.redactedAndTruncated() ?? "<nil>"])
```

Comment:
Perhaps its worth to implement custom interpolation to reduce the duplication in the log attributes and also to enforce  redacted content.